package/etc/conf.d/conflib/syslog/app-syslog-barracuda_waf.conf

```diff
@@ -3,16 +3,54 @@ block parser app-syslog-barracuda_waf() {
         rewrite {
             r_set_splunk_dest_default(
                 index("netwaf")
-                sourcetype('barracuda:waf')
                 vendor("barracuda")
                 product("syslog")
+                sourcetype('barracuda:syslog')
             )
         };
+
+        if (program('SYS')) {
+            rewrite {
+                r_set_splunk_dest_update_v2(
+                    sourcetype('barracuda:system')
+                );
+            };
+        } elif (program('WF')) {
+            rewrite {
+                r_set_splunk_dest_update_v2(
+                    sourcetype('barracuda:waf')
+                );
+            };   
+        } elif (program('TR')) {
+            rewrite {
+                r_set_splunk_dest_update_v2(
+                    sourcetype('barracuda:web')
+                );
+            };   
+        } elif (program('AUDIT')) {
+            rewrite {
+                r_set_splunk_dest_update_v2(
+                    sourcetype('barracuda:audit')
+                );
+            };   
+        } elif (program('NF')) {
+            rewrite {
+                r_set_splunk_dest_update_v2(
+                    sourcetype('barracuda:firewall')
+                );
+            };   
+        };
     };
 };
+
+
 application app-syslog-barracuda_waf[sc4s-syslog] {
     filter {
-        program('waf-den'  type(string)  flags(prefix));
+        program('SYS' type(string))
```

Comment:
may be we can use ignore-case and prefix flags here

Reply:
added prefixes, skipped `or` in the first condition

Reply:
removed both ignorecase and prefix flags because of failing tests (disrupting other test cases due to too broad filters)